gatorgrade/output/src/output_functions.py

```diff
@@ -0,0 +1,69 @@
+"""
+This class is used for storing the main functions requested from the Github.

```

Comment:
This should be changed to what is located in the command-receive-function branch so that there are less merge errors when we combine them.

The docstring located in that branch is:

```
"""This module is used for storing the main functions requested.

The requested functions are located at the Github Issue Tracker
for the output team. For instance, functions dealing with percentage
output, description output, and colorization of text.
"""
```